### ISSUE TITLE
Why does this extension require access to all webpage data?

### ISSUE THREAD

**Issue by Zaijo** [marioestrada/safari-omnikey#29]
<img width="529" alt="screen shot 2018-06-05 at 14 58 24" src="https://user-images.githubusercontent.com/1964908/40977409-2771d392-68d1-11e8-8c1a-6f0ffb1012b6.png">

**Comment by Zaijo** [comment]
Anyway, thanks for a great job, but I'd like to see it run without this "access everywhere" feature. Is it possible?

**Comment by sebknzl** [comment]
Yes, I'd also like to know! Thanks

**Comment by Zaijo** [comment]
There is a workaround using "Show extension builder" and switching the "access all" property. Then, unless you are an Apple Developer, you can use the modified extension "until you quit Safari". Better than nothing.

**Comment by ksperling** [comment]
Yes, it would definitely be good to add an explanation for this to the FAQ
